src/main/java/task/TaskList.java

```diff
@@ -0,0 +1,110 @@
+package task;
+
+public class TaskList {
+    private final Task[] taskList;
+    private int taskCount = 0;
+
+    public TaskList(int size) {
+        taskList = new Task[size];
+    }
+
+    public void printTasks() {
+        System.out.println("Take a look, all the tasks you have here, so many, yuuuuuuge\n");
+        for (int i = 0; i < taskCount; i++) {
+            int temp = i + 1;
+            System.out.println(temp + ". " + taskList[i].printTask());
+        }
+    }
+
+    public Task getTask(int id) {
+        return taskList[id];
+    }
+
+    public int getTaskCount() {
+        return taskCount;
+    }
+
+    public void deleteTask(int taskNumber) {
+        Task tempTask = taskList[taskNumber];
```

Comment:
tempTask is used to refer to a task temporarily during deletion or other operations, very well used!